service/history/historyEngine.go

```diff
@@ -322,6 +322,7 @@ Update_History_Loop:
 		if err0 != nil {
 			return nil, err0
 		}
+		tBuilder := e.getTimerBuilder(&context.workflowExecution)
```

Comment:
The historyengine file uses this name in other places. so just left the same consistent with it. rename can be done at some time, as a separate CR. the renames overshadows other changes.